src/systems/diff_drive/DiffDrive.hh

```diff
@@ -61,7 +61,23 @@ namespace systems
   /// `<odom_topic>`: Custom topic on which this system will publish odometry
   /// messages. This element if optional, and the default value is
   /// `/model/{name_of_model}/odometry`.
-  class DiffDrive
+  ///
+  /// `<tf_topic>`: Custom topic on which this system will publish the
+  /// transform from `frame_id` to `child_frame_id`. This element if optional,
+  ///  and the default value is `/model/{name_of_model}/tf`.
+  ///
+  /// `<frame_id>`: Custom `frame_id` field that this system will use as the
+  /// origin of the odometry transform in both the `<tf_topic>`
+  /// `ignition.msgs.Pose_V` message and the `<odom_topic>`
+  /// `ignition.msgs.Odometry` message. This element if optional, and the
+  /// default value is `{name_of_model}/odom`.
+  ///
+  /// `<child_frame_id>`: Custom `child_frame_id` that this system will use as
+  /// the target of the odometry trasnform in both the `<tf_topic>`
+  /// `ignition.msgs.Pose_V` message and the `<odom_topic>`
+  /// `ignition.msgs.Odometry` message. This element if optional,
+  ///  and the default value is `{name_of_model}/{name_of_link}`.
+  class IGNITION_GAZEBO_VISIBLE DiffDrive
```

Comment:
@chapulina , I merged in the lastest ign-gazebo4 code and there was a conflict here. Specifically, the `DiffDrive` class in Edifice lacked `IGNITION_GAZEBO_VISIBLE`. This seemed strange. I made this change, but I'm not sure if I missed some prior important change that caused this.

Reply:
As part of the fixes for Windows, we've been removing the visibility flags for system headers, because they're not installed. See some context here: https://github.com/ignitionrobotics/ign-gazebo/pull/501#discussion_r555462885

I see this PR needs conflict resolution again. I'll address this while I do that.